packages.json

```diff
@@ -31595,5 +31595,18 @@
     "description": "An unofficial wrapper to Hyprland's IPC layer",
     "license": "GPLv3",
     "web": "https://github.com/xTrayambak/hyprland_ipc"
+  },
+  {
+    "name": "gemmaJSON",
+    "url": "https://github.com/sainttttt/gemmaJSON",
+    "method": "git",
+    "tags": [
+      "\u001B[15;2~simd",
```

Comment:
Is it a typo?

Reply:
Yeah! Not sure why that showed up, I just used nimble publish, but somehow that showed up. I re-did nimble publish and it created a new PR, so I guess this can be closed.